src/controllers/TalksController.php

```diff
@@ -51,13 +51,11 @@ public function getAction($request, $db)
 
     public function postAction($request, $db)
     {
-        if (! isset($request->user_id)) {
-            throw new Exception("You must be logged in to create data", 401);
-        }
+        $this->checkLoggedIn($request);
```

Comment:
I'd prefer to have those refactorings in a separate PR. They aren't necessary for the feature. Having them in two separate PRs makes each one easier to digest :)